pypeit/core/extract.py

```diff
@@ -2386,3 +2387,174 @@ def ech_objfind(image, ivar, slitmask, slit_left, slit_righ, order_vec, maskslit
 
     return sobjs_final, skymask[allmask]
 
+
+class ManualExtractionObj(pypeitpar.ParSet):
```

Comment:
Two suggestions:
 - I would make this a `DataContainer` instead of a `ParSet`.
 - I don't like this class being in a core module.  I would make it its own top-level module, or, e.g., put it in `Reduce`.